### PR TITLE
version push for 2017 update 1

### PR DESCRIPTION
**Pull request by jhein32**
Easy config for next update.  Still need to check the vtune works on our cluster.

### CODE REVIEW

File: easybuild/easyconfigs/v/VTune/VTune-2017_update1.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+license_file = HOME + '/licenses/intel/license.lic'
+
+moduleclass = 'tools'
+
+
+
+
+
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/v/VTune/VTune-2017_update1.eb]: @jhein32 please trim the trailing whitespace...

**Comment by jhein32** [comment]
Hi Kenneth

Clean-up as requested.  I also added an md5 checksum to the advisor (I was working on that as well, not seeing that there had been a version in the then develop).  I hope you don't mind so much, bundling that in here.  

Best wishes
  Joachim

**Comment by jhein32** [comment]
Is here something left to be done or should this be merged/marked as milestone 3.0.2?

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2684.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ca02cb76d79d62786c304fc8659dd2fa for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2082.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f4022e407bdb403714e4c354a1137b74 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jhein32!

**Pull request merged**